c/driver/sqlite/sqlite_test.cc

```diff
@@ -39,8 +39,7 @@ class SqliteQuirks : public adbc_validation::DriverQuirks {
   AdbcStatusCode SetupDatabase(struct AdbcDatabase* database,
                                struct AdbcError* error) const override {
     // Shared DB required for transaction tests
-    return AdbcDatabaseSetOption(
-        database, "uri", "file:Sqlite_Transactions?mode=memory&cache=shared", error);
+    return AdbcDatabaseSetOption(database, "uri", "file:Sqlite_Transactions", error);
```

Comment:
Hmm, why was this changed? This is specifically to make sure two connections in the same process can share the same underlying database

Reply:
Mistake - just wanted to get a file for local testing